paimon-spark/paimon-spark-ut/src/test/scala/org/apache/paimon/spark/sql/InsertOverwriteTableTestBase.scala

```diff
@@ -570,7 +570,6 @@ abstract class InsertOverwriteTableTestBase extends PaimonSparkTestBase {
           sql(
             "INSERT INTO t VALUES (1, STRUCT(1, 1)), (2, null), (3, STRUCT(1, null)), (4, STRUCT(null, null))")
           if (format.equals("parquet")) {
-            // todo: fix it, see https://github.com/apache/paimon/issues/4785
```

Comment:
Oh, no, This bug has not been fixed. Please read carefully: https://github.com/apache/paimon/issues/4785.

Reply:
After the fix, we do not need this if-else statement..

Reply:
Thanks for your review! Sorry for the inconvenience. Can we reopen it?

Reply:
Oh, it’s not necessary, we will start fixing it soon